packages/teams-js/src/internal/interfaces.ts

```diff
@@ -50,3 +52,19 @@ export interface DOMMessageEvent {
   func: string;
   args?: any[];
 }
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * UUID object
+ */
+export class UUID {
+  public constructor(private readonly uuid: string = generateGUID()) {
+    validateUuid(uuid);
+  }
+
+  public toString(): string {
+    return this.uuid;
+  }
+}
```

Comment:
Thanks for putting this here and enabling easy shared usage! Can you write a small set of unit tests to validate its functionality (e.g. that it doesn't accept invalid uuids as input, etc.)?